packages/flutter/lib/src/services/text_formatter.dart

```diff
@@ -4,38 +4,38 @@
 
 import 'package:flutter/services.dart';
 
-/// A [TextInputFormatter] can be optionally injected into an [EditableText] 
+/// A [TextInputFormatter] can be optionally injected into an [EditableText]
 /// to provide as-you-type validation and formatting of the text being edited.
-/// 
+///
 /// Text modification should only be applied when text is being committed by the
-/// IME and not on text under composition (i.e. when 
-/// [TextEditingValue.composing] is collapsed). 
-/// 
-/// Concrete implementations [BlacklistingTextInputFormatter], which removes 
-/// blacklisted characters upon edit commit, and 
+/// IME and not on text under composition (i.e. when
```

Comment:
`i.e. only when`... to make it clear the i.e. applies to the first clause and not the second.